mern/client/src/Components/Navbar/navbar.component.js

```diff
@@ -7,17 +7,15 @@ import { SessionContext } from "../../Contexts/sessionProvider";
 
 export default function Navbar() {
   // define context var to show/hide nav buttons
-  const [navState, setNavState] = useContext(SessionContext)
-  
-  // Create session id var and setter function
-  const sessionid = "12345";
-  // const setSessionID = (id) => {
-  //   sessionid = id;
-  // };
+  const [sessionID, setSessionID, , ] = useContext(SessionContext)
 
   return (
     <nav className="navbar-bg">
-      <Link to="/" className="nav-link" onClick={() => {setNavState(false)}}>
+      <Link to="/" className="nav-link" onClick={() => {
+        setSessionID(false)
+        // reset sessionID on return to home page
+        setSessionID()
```

Comment:
hmm wait why do you set it as false and then as empty?

Reply:
Ah thank you for the catch! The set false was from when we had navState, but that's outdated now!